include/pcl_conversions/pcl_conversions.h

```diff
@@ -39,33 +39,32 @@
 
 #include <vector>
 
-#include <ros/ros.h>
-
 #include <pcl/conversions.h>
 
 #include <pcl/PCLHeader.h>
-#include <std_msgs/Header.h>
+#include <std_msgs/msg/header.hpp>
 
 #include <pcl/PCLImage.h>
-#include <sensor_msgs/Image.h>
+#include <sensor_msgs/msg/image.hpp>
 
 #include <pcl/PCLPointField.h>
-#include <sensor_msgs/PointField.h>
+#include <sensor_msgs/msg/point_field.hpp>
 
 #include <pcl/PCLPointCloud2.h>
-#include <sensor_msgs/PointCloud2.h>
+#include <sensor_msgs/msg/point_cloud2.hpp>
 
-#include <pcl/PointIndices.h>
-#include <pcl_msgs/PointIndices.h>
+// TODO(clalancette): Re-enable once we have pcl_msgs
+//#include <pcl/PointIndices.h>
+//#include <pcl_msgs/PointIndices.h>
 
-#include <pcl/ModelCoefficients.h>
-#include <pcl_msgs/ModelCoefficients.h>
+//#include <pcl/ModelCoefficients.h>
```

Comment:
Technically, I can leave in the pcl/ ones.  Those should work.  They just aren't necessary/useful until we uncomment the pcl_msgs versions, and I think it is a little cleaner since the TODO comment applies to the whole block.  That being said, I don't care all that much, so if you think I should uncomment them, I will.  Just let me know.